Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -673,6 +673,12 @@ open class LottieAnimationView: LottieAnimationViewBase {
     lottieAnimationLayer.setValueProvider(valueProvider, keypath: keypath)
   }
 
+  /// Sets a ValueProvider for the specified keypath. The value provider will be removed
+  /// on all properties that match the keypath.
+  public func removeValueProvider(keypath: AnimationKeypath) {
```

Comment:
A better name for this would probably be:

```suggestion
  public func removeValueProvider(for keypath: AnimationKeypath) {
```